micromamba/src/constructor.cpp

```diff
@@ -97,6 +97,7 @@ construct(const fs::path& prefix, bool extract_conda_pkgs, bool extract_tarball)
                 std::string pkg_name = index["name"];
 
                 index["fn"] = entry.path().filename();
+                bool found_match = false;
```

Comment:
Is this also part of the other PR?

Reply:
It's mostly there as it would have made debugging https://github.com/conda/constructor/pull/487 much easier.

This can be either PR or forgotten about entirely, as you prefer.